doc/review-assignments.md

```diff
@@ -0,0 +1,30 @@
+# Table Flip Review Assignment Process
+
+## cloud-init
+
+At stand-up each day, team will assign any new pull requests for
+review.  This assignment is done algorithmically, to simplify matters:
+
+* New pull requests are assigned from lowest number to highest (i.e.
+  older first)
+* The order in which pull requests assignees are selected is determined
+  by the daily triage order, starting from whoever is on triage duty
+  today
+    * Currently that order is: Dan, Ryan, Chad
+    * For example, on a Wednesday (Ryan's triage day), the first PR
+      would be assigned to Ryan, the second to Chad, and the third to
+      Dan
+* On a day where a reviewer isn't on triage, the next triager who is a
+  reviewer will be considered to be "first"
+    * For example, on Fridays, whoever is on rotating Monday triage
+      will be assigned the first PR.
+* When a reviewer is assigned to a PR, this means they should be a
+  literal Assignee on the pull request in GitHub terms
```

Comment:
Should we describe that the role of the Assignee means that they are responsible for initial review,  following up on the PR if stale and requesting additional reviews if needed to get this PR landed?

Reply:
we can refine this aspect later after a couple of iterations during standup if needed.